File: PRAKTIKUM I/Modul 6/Python/PRAK601-2310817110015-AKHMAD_CHAIDAR_ANANDA.py
# fungsi untuk menampilkan matrix
def tampilkan_matrix(baris, kolom):
    # input matrix
    a = [int(i) for i in input().split()]
    
    print()
    # output matrix
    for i in range(baris * kolom):
        print(a[i], end=' ')
        # kondisi jika index terakhir dari matrix
        # mencapai kolom terakhir dari baris matrix
        if (i + 1) % kolom == 0: print()

# input
baris, kolom = map(int, input().split())

# pemanggilan fungsi matrix
tampilkan_matrix(baris, kolom)

File: PRAKTIKUM I/Modul 6/C/PRAK601-2310817110015-AKHMAD_CHAIDAR_ANANDA.c
#include<stdio.h>

// fungsi untuk menampilkan matrix
void tampilkan_matrix(int baris, int kolom)
{
    // deklarasi variabel array matrix
    int matrix[baris][kolom];
    // deklarasi variabel perulangan
    int i, j;

    // perulangan matrix 
    for (i = 0; i < baris; i++){
        for (j = 0; j < kolom; j++){
            scanf("%d", &matrix[i][j]);
        }
    }
    printf("\n");
    // output matrix
    for (i = 0; i < baris; i++){
        for (j = 0; j < kolom; j++){
            printf("%d ", matrix[i][j]);
        }
        printf("\n");
    }
}

int main(){
    // deklarasi variabel
    int baris, kolom;
    // input 
    scanf("%d %d", &baris, &kolom);

    // memanggil fungsi matrix
    tampilkan_matrix(baris, kolom);

    return 0;
}